packages/radix-ui-themes/src/components/link.css

```diff
@@ -45,6 +45,13 @@
   text-underline-offset: calc(0.025em + 2px);
   text-decoration-color: var(--accent-a5);
 
+  /* Enhancement for bright colors */
+  @supports (background: color-mix(in oklab, white, black)) {
+    & {
+      text-decoration-color: color-mix(in lch, var(--accent-a5), var(--gray-a6));
```

Comment:
we're testing for oklab support but then using lch? is that intentional? I see below we're using oklab

Reply:
Not intentional, thanks for the spot